### PR TITLE
Remove feature flag: replace 'COMPLETED' with 'DONE' from 'updateTaskStatus' function

### PR DESCRIPTION
**Pull request by shubhamsigdar1**
## Issue Ticket Number:-  https://github.com/Real-Dev-Squad/website-backend/issues/1627
Pr depend on- https://github.com/Real-Dev-Squad/website-my/issues/521

## Description: 

- remove COMPLETED with 'DONE' from 'updateTaskStatus' function
- Now user change status to DONE then all the condition before applied to COMPLETED will replace by DONE
- Under feature flag changes PR which merged before: https://github.com/Real-Dev-Squad/website-backend/pull/1657

Is Under Feature Flag 
- [ ] Yes
- [X] No

Database changes
- [ ] Yes
- [x] No

Breaking changes (If your feature is breaking/missing something please mention pending tickets)
- [ ] Yes
- [x] No

Is Development Tested?

- [x] Yes
- [ ] No

### Images/video of the change:

**Before Changes**

https://github.com/Real-Dev-Squad/website-my/assets/107163260/31c3d80a-d25e-499c-bc7c-52f0dfeac180

**After Changes**

https://github.com/Real-Dev-Squad/website-my/assets/107163260/e1105d30-b94b-41a7-8eae-6add5afeda89

### CODE REVIEW

File: controllers/tasks.js
Comment (heyrandhir) [inline controllers/tasks.js:1]: Shouldn't we run the script post your one time script runs successfully ? have it been ran by Ankush ?

**Review by heyrandhir — CHANGES_REQUESTED** [review]

**Comment by heyrandhir** [comment]
At 00:08 sec in the after video it gave a error while you manually chose `Done` from the menu then it changed when you moved the slider to 100. 

image

**Comment by shubhamsigdar1** [comment]
> At 00:08 sec in the after video it gave a error while you manually chose `Done` from the menu then it changed when you moved the slider to 100.
> 
> image

Yes, because I am trying to change status directly to DONE and progress of task is not 100%. So we cannot update DONE directly

File: controllers/tasks.js
Comment (shubhamsigdar1) [reply controllers/tasks.js:1]: No, Ankush Didn't run it. 
This pr get merged only after ankush will run script

**Review by ajoykumardas12 — APPROVED** [review]

**Pull request merged**